straxen/plugins/peaklet_processing.py

```diff
@@ -17,12 +17,21 @@
                  help="Include this many ns right of hits in peaks"),
     strax.Option('peak_min_pmts', default=2,
                  help="Minimum contributing PMTs needed to define a peak"),
-    strax.Option('peaklet_split_min_height', default=25,
-                 help="Minimum height in PE above a local sum waveform"
-                      "minimum, on either side, to trigger a split"),
-    strax.Option('peaklet_split_min_ratio', default=4,
-                 help="Minimum ratio between local sum waveform"
-                      "minimum and maxima on either side, to trigger a split"),
+    strax.Option('peak_split_gof_threshold',
+                 default=(
+                     None,  # Reserved
+                     ((0.5, 1), (3.5, 0.25)),
+                     ((2, 1), (4.5, 0.4))),
+                 help='Natural breaks goodness of fit/split threshold to split '
```

Comment:
A comment pointing one to your note might help future maintenance.

Reply:
Thanks, added one a few lines above